integration/testdata/source_2.1_explicit_runtime/netcoreapp2.csproj

```diff
@@ -1,16 +0,0 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
-
-  <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
-    <RuntimeFrameworkVersion>2.1.0</RuntimeFrameworkVersion>
```

Comment:
Seems like the crux of the versioning test is the difference between this (explicit) version and the "floating" runtime version specified [in the other fixture's csproj](https://github.com/paketo-buildpacks/dotnet-publish/blob/c49c431209299a84536713b40a2ac1bb0b6d8708/integration/testdata/source_2.1_float_runtime/netcoreapp2.csproj#L5). 

This difference seems related to [this issue](https://github.com/paketo-buildpacks/dotnet-core-runtime/issues/135) about roll-forward logic. But the tests here don't make any assertions about the runtime version installed. [We have tests in runtime](https://github.com/paketo-buildpacks/dotnet-core-runtime/blob/cec9c25e5ce73128d3f600809610a90f9c475959/runtime_version_resolver_test.go#L183) that figure out the proper version to install given constraints from the projfile. I agree, don't think we need to test this here.